### ISSUE TITLE
1.3 and ie

### ISSUE THREAD

**Issue by darkstar2002** [craigk5n/webcalendar#126]
Did a test upgrade to 1.3 from 1.2.8 due to us having to use php7 soon.   Most of the layout is missing when viewed from IE.  It looks fine in firefox and chrome.

**Comment by craigk5n** [comment]
Yep, I tried this with IE11, and it looks terrible.  I tried it with MS Edge and it looks correct.  Looks like IE still has 10% of the browser market, so this is something that should get fixed.

**Comment by FranzGotsis** [comment]
Sadly Edge does have some issues as well. I have a user complaining that he cannot easily edit fields in forms (but he found a workaround) on a windows phone.
I am using WebCalendar 1.2.9 with many browsers including IE without problems, so there must be a recent change in WebCalendar 1.3.0 that broke it.

**Comment by codifex** [comment]
The problem seems to still exist in v1.3.0 on IE11.
